tiles-generation/cdk/lambda/email-notification.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * Lambda function for tiles generation ECS task state change email notification.
+ */
+
+const aws = require('aws-sdk');
+const sns = new aws.SNS();
+const eventProcess = require('./event-process');
+
+exports.handler = function(event, context, callback) {
+    try {
+        const eventText = JSON.stringify(event, null, 2);
+
+        console.log(`\n${eventText}`);
```

Comment:
does this log contain any sensitive information which should be masked?

Reply:
The log contains information about the ECS task state change like [this](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_cwe_events.html#ecs_task_events) , but the logs only printed on internal Lambda logs on CloudWatch. The reason I put here is that it will help debug and analyze for lambda execution.

Reply:
Yeah so when we have details like email, ip addresses, we usually mask those details even in logs. But depends on the usecase.

Reply:
That make sense, but here won't have this kind of sensitive information.